src/pages/contributors/subcomponents/ContributorsDetails.tsx

```diff
@@ -43,6 +43,8 @@ class ContributorsDetails extends React.Component {
               <br />
               Thomas Tan
               <br />
+              Thomas Tan
+              <br />
```

Comment:
This seems like a merging accident

Reply:
Oh, that one: I've fixed a small issue with formatting the names yesterday, and this is the merged version. Previously Chrome was splitting one of the names for some strange reason.

Reply:
The change seems to be included as part of the base already (see the [full diff](https://github.com/source-academy/cadet-frontend/pull/1651/files#diff-abff1c056c5a85449bb167175449597666e77d510b735783ed24aa6e2fead769R46))&mdash;this is adding a duplicate "Thomas Tan")